src/components/Clock.jsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export class Clock extends React.Component {
+  state = {
+    clock: new Date().toLocaleTimeString(),
+  }
+
+  componentDidMount() {
+    this.setState({
+      time: setInterval(() => {
+        this.setState({ clock: new Date().toLocaleTimeString() });
+        //  eslint-disable-next-line
+        console.log(this.state.clock);
+      }, 1000),
```

Comment:
```suggestion
    this.time = setInterval(() => {
        this.setState({ clock: new Date().toLocaleTimeString() });
        //  eslint-disable-next-line
        console.log(this.state.clock);
      }, 1000),
```
It's not necessary to write setInterval to the state